src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useRef, useState} from "react";
+import styled from "styled-components";
+import BREAKPOINT from "../../../variables/breakpoint";
+import COLOR from "../../../variables/color"
+import FONT_FAMILY from "../../../variables/font_family"
+import TEXTS from "../../../variables/texts"
+
+const Input = ({onEditComplete, defaultValue}) => {
+  const inputRef = useRef(null);
+
+  const onEnterPush = (e) => {
+    if (e.key === "Enter") {
+      inputRef.current.blur();
+      inputRef.current.removeEventListener("blur", () => {onEditComplete(inputRef.current.value);});
+    }
+  };
```

Comment:
「押下」は `press` が適切な気がします！
（ [`keypress` イベント](https://developer.mozilla.org/ja/docs/Web/API/Element/keypress_event)は非推奨になっているので、そことの差別化の観点では現状の命名でも問題ないと考えることもできます ）

```suggestion
  const onPressEnter = (e) => {
    if (e.key === "Enter") {
      inputRef.current.blur();
      inputRef.current.removeEventListener("blur", () => {onEditComplete(inputRef.current.value);});
    }
  };
```